.github/workflows/olm_pr.yaml

```diff
@@ -69,7 +70,7 @@ jobs:
           push-to-fork: k8gb-io/community-operators
           path: sandbox
           commit-message: OLM bundle for k8gb@${{ steps.get_version.outputs.bundleDir }}
-          title: OLM bundle for k8gb@${{ steps.get_version.outputs.bundleDir }}
+          title: operators k8gb ({{ steps.get_version.outputs.bundleDir }})
```

Comment:
why this title change?

Reply:
it's not necessary, but every time we create this PR with this title, it gets automatically [renamed](https://github.com/k8s-operatorhub/community-operators/pull/356) to this form
<img width="827" alt="Screenshot 2021-11-01 at 12 11 41" src="https://user-images.githubusercontent.com/535866/139662985-d4d1642c-be4f-4b57-930c-103fd4cc145f.png">